doc/source/serve/doc_code/hpu_inference_serve_deepspeed.py

```diff
@@ -0,0 +1,265 @@
+# __worker_def_start__
+import tempfile
+from typing import Dict, Any
+from starlette.responses import Request, StreamingResponse
+
+import torch
+from transformers import TextStreamer
+
+import ray
+from ray import serve
+from ray.util.queue import Queue
+from ray.runtime_env import RuntimeEnv
+
+
+@ray.remote(num_cpus=8, resources={"HPU": 1})
```

Comment:
Why 8 CPUs? Could you explain in a comment?

Reply:
It was a random number. I have removed it.